src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -64,7 +64,9 @@ const TutorProfile = () => {
     <PageWrapper>
       <ProfileInfo myRole={userRole} userData={response} />
       <CompleteProfileBlock data={response} profileItems={profileItems} />
-      <AboutTutorBlock />
+      {response.professionalBlock && (
+        <AboutTutorBlock data={response.professionalBlock} />
+      )}
```

Comment:
Also here. Think about move it before return statment to increase readability

Reply:
In future we should separate this logic to different file, because it's tutor related only, so i think it's okay if it stays here